scripts/mods/mod_reforged/perk_groups/pg_rf_large.nut

```diff
@@ -21,7 +21,7 @@ this.pg_rf_large <- ::inherit(::DynamicPerks.Class.PerkGroup, {
 				[],
 				[],
 				[],
-				[],
+				["perk.rf_second_wind"],
```

Comment:
Don't add the perk to the perk group in the perk's implementation PR. We'll make a separate PR later where each perk group is fully updated with all the perks it needs.